Gemfile.lock

```diff
@@ -210,35 +209,47 @@ GEM
       rails (>= 3.0)
     globalid (0.3.6)
       activesupport (>= 4.1.0)
-    google-api-client (0.7.1)
-      addressable (>= 2.3.2)
-      autoparse (>= 0.3.3)
-      extlib (>= 0.9.15)
-      faraday (>= 0.9.0)
-      jwt (>= 0.1.5)
-      launchy (>= 2.1.1)
-      multi_json (>= 1.0.0)
-      retriable (>= 1.4)
-      signet (>= 0.5.0)
-      uuidtools (>= 2.1.0)
-    guard (2.6.1)
+    google-api-client (0.8.6)
```

Comment:
There's enough change in this gemfile that we should test this PR in production and make sure everything works.  ... or you can reduce the Gemfile diff if you'd like.